cps/web.py

```diff
@@ -47,12 +47,20 @@
 from shutil import move, copyfile
 from tornado.ioloop import IOLoop
 from tornado import version as tornadoVersion
+#from builtins import str
+
+try:
+    from urllib.parse import quote   
+    from imp import reload
+    from past.builtins import xrange
+except ImportError as e:
+    from urllib import quote
 
```

Comment:
Python3 moval
reload  to imp.reload
urllib.quote to urllib.parse.quote
no xrange , need pip install future , also can refactor out xrange later